theme_cobalt/views/customizations.xml

```diff
@@ -5,7 +5,11 @@
 <template id="s_banner" inherit_id="website.s_banner" name="Cobalt s_banner">
     <!-- Content -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_cc o_cc2 pt152 pb128" remove="pt96 pb96" separator=" "/>
+        <attribute name="class" add="pt24 pb80" remove="pt96 pb96" separator=" "/>
+        <attribute name="data-oe-shape-data">{'shape':'web_editor/Origins/04_001','flip':['y'],'showOnMobile':false,'shapeAnimationSpeed':'0'}</attribute>
+    </xpath>
+    <xpath expr="//div[hasclass('container')]" position="before">
+        <div class="o_we_shape o_web_editor_Origins_04_001" style="background-image: url('/web_editor/shape/web_editor/Origins/04_001.svg?c3=o-color-3&amp;flip=y');"/>
```

Comment:
You should also probably modify the `new_page_template.xml` file :+1: 
Indeed, now, if you add a new page with a banner (e.g. The last one of the 'About' category), you have something like :
```.xml
<section>
     <div class="o_web_editor_Origins_11_001"/>
     <div class="o_web_editor_Origins_04_001"/>
<section/>
```